Explorer/Explorer.sln.DotSettings

```diff
@@ -518,5 +518,6 @@
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Prewarmed/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Reloader/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=screencapture/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=teleporter/@EntryIndexedValue">True</s:Boolean>
```

Comment:
Whats this? 🤔

Reply:
It adds "teleporter" to the project dictionary so it's not marked as misspelled. I like adding these so I don't get the annoying green squiggly line in Rider 😄